### ISSUE TITLE
Tasks run in the current directory by default

### ISSUE THREAD

**Issue by dbohdan** [nat-n/poethepoet#180]
In version 0.24.2 (since commit https://github.com/nat-n/poethepoet/commit/51a6fd7953e5b8b85babe37eacc7225a7626b06b? I haven't bisected it.) the default task behavior has changed. Tasks run in the current directory instead of the project directory. This looks like a bug, because the [documentation](https://poethepoet.natn.io/tasks/options.html#running-a-task-with-a-specific-working-directory) says:

> By default tasks are run from the project root – that is the parent directory of the pyproject.toml file. 

The change is the easiest to demonstrate if you define a task that prints the current directory.

```
[tool.poe.tasks.pwd]
cmd = "pwd"
```

Then run,

```none
> pipx install poethepoet==0.24.1
...
> poe --root proj pwd
Poe => pwd
/home/user/proj
> pipx uninstall poethepoet
...
> pipx install poethepoet==0.24.2
...
> poe --root proj pwd
Poe => pwd
/home/user
```

As a temporary **workaround**, you can add `cwd = ""` to your tasks.

**Comment by nat-n** [comment]
Hi @dbohdan, thanks for reporting this 🙂
Indeed this bug probably slipped in as part of [this change](https://github.com/nat-n/poethepoet/commit/51a6fd7953e5b8b85babe37eacc7225a7626b06b) although I'm not sure exactly how, and it seems something about how I test most features (running code from the repo with poe) stops it from manifesting.

I'm working on a fix.

**Comment by nat-n** [comment]
Aaaaaaaaaand it's fixed [0.24.3](https://pypi.org/project/poethepoet/0.24.3/) 🚀

**Comment by dbohdan** [comment]
Yay!
